src/cascadia/TerminalApp/App.cpp

```diff
@@ -828,6 +811,8 @@ namespace winrt::TerminalApp::implementation
             "TabInformation",
             TraceLoggingDescription("Event emitted upon new tab creation in TerminalApp"),
             TraceLoggingInt32(tabCount, "TabCount", "Count of tabs curently opened in TerminalApp"),
+            TraceLoggingBool(profileIndex.has_value(), "ProfileSpecified", "Whether the new tab specified a profile explicitly"),

```

Comment:
wait a profile's always going to have a guid, right? Even if the profile didn't have a GUID, we'll generate one for them, or at this point it'll just be {0}, right?

Reply:
it'll always have a guid